src/envs/behavior.py

```diff
@@ -435,24 +435,26 @@ def make_behavior_option(name: str, types: Sequence[Type], params_space: Box,
     def _policy(state: State, memory: Dict, _objects: Sequence[Object],
                 _params: Array) -> Action:
         assert "has_terminated" in memory
-        assert ("controller" in memory and memory["controller"] is not None
+        assert ("policy_controller" in memory
+                and memory["policy_controller"] is not None
                 )  # must call initiable() first, and it must return True
         assert not memory["has_terminated"]
-        action_arr, memory["has_terminated"] = memory["controller"](state, env)
+        action_arr, memory["has_terminated"] = memory["policy_controller"](
+            state, env)
         return Action(action_arr)
 
     def _initiable(state: State, memory: Dict, objects: Sequence[Object],
                    params: Array) -> bool:
         igo = [object_to_ig_object(o) for o in objects]
         assert len(igo) == 1
-        if memory.get("controller") is None:
+        if memory.get("policy_controller") is None:
             # We want to reset the state of the environment to
             # the state in the init state so that our options can
             # run RRT/plan from here as intended!
             if state.simulator_state is not None:
                 env.task.reset_scene(state.simulator_state)
             controller = controller_fn(env, igo[0], params, rng=rng)
-            memory["controller"] = controller
+            memory["policy_controller"] = controller
```

Comment:
you'll need to add memory_controller here too with your bit being True

Reply:
right, but that's not implemented yet, so i wasn't sure whether to include this in this PR

Reply:
should i add the option model code to this PR as well? Or should i open a separate PR that includes that stuff in `behavior_options.py` first and merge that first?

Reply:
i'd say add it to this PR. but you can do the changing of the argument type from iGibsonEnv to BehaviorEnv in a separate PR